packages/element/src/index.native.js

```diff
@@ -38,3 +38,22 @@ export { Component };
  * @return {string} HTML.
  */
 export { serialize as renderToString };
+
+/**
+ * Component used as equivalent of Fragment with unescaped HTML, in cases where
+ * it is desirable to render dangerous HTML without needing a wrapper element.
+ * To preserve additional props, a `div` wrapper _will_ be created if any props
+ * aside from `children` are passed.
+ *
+ * @param {string} props.children HTML to render.
+ *
+ * @return {WPElement} Dangerously-rendering element.
+ */
+export function RawHTML( { children, ...props } ) {
```

Comment:
Can we move this component to its own file (`raw-hatml.js`?) and reference it from both index files to avoid code duplication and possible sync issues in the future?

Reply:
Good call @gziolo , made the change with da43e07.

I chose the rather boring name `index-common.js` for the base module, also having in mind that more functions will most probably end up there as we expose them to the native app. Let me know if you think the name can be a problem.